Makefile

```diff
@@ -32,6 +35,12 @@ test:
 	go vet -mod=readonly  ./...
 	go test -mod=readonly -race ./...
 
+lint:
+ifndef LINT
+	curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(shell go env GOPATH)/bin v1.17.1
```

Comment:
I think we should never promote running scripts from the internet. How about including `golangci-lint.sh` in our repository instead?

Reply:
that's fair, this can also be pointed at github stuff, but otherwise I agree that we can clone the script to control it.

Reply:
I agree with copying the install script at least. 
Still download binary, but seems safer than random shell scripts